packages/local_testnet/Readme.md

```diff
@@ -153,19 +173,20 @@ After deployment, note down the addresses for the following contracts:
    BONSAI_API_URL="your_api_url"
```

Comment:
Under "Deploy the RISC Zero Contracts".
Step 2 & 3 are optional. But its recommended to run it using bonsai for faster compute.
https://dev.risczero.com/api/generating-proofs/remote-proving
Maybe we can link to this doc and remove these steps?
Wdy think? @0xjei

Reply:
IMHO we can keep the steps because the Bonsai documentation does not replace the "how to config" in CRISP (it says to set the environment variables, but not where), and at the same time clearly state that these are optional. wdyt?

Reply:
see [6045611](https://github.com/gnosisguild/CRISP/pull/29/commits/6045611cae0e65f3d059abf58bd1f61422f85042)

Reply:
Thats much better!